revscoring/features/diff.py

```diff
@@ -133,10 +131,13 @@ def process_uppercase_chars_removed(diff_removed_segments):
         modifiers.max(parent_revision.proportion_of_uppercase_chars, 1)
 
 def process_longest_repeated_char_added(diff_added_segments):
-    return      max(sum(1 for _ in group)
-                    for segment in diff_added_segments
-                    for _, group in groupby(segment.lower()))
-
+    try:
+        return max(sum(1 for _ in group)
+                   for segment in diff_added_segments
+                   for _, group in groupby(segment.lower()))
+    except ValueError:
+        # Happens when there's no segments added
+        return 0
```

Comment:
Either value is sort of nonsensical.  I suppose that in the cases where there actually was added content, the min expected value is 1.  I don't have a strong opinion either way.  It seems like you'd prefer 1, so I'll make the change.